fern/docs/pages/docs/writing-content.mdx

```diff
@@ -1,10 +1,80 @@
+
+### Frontmatter
+
+You can optionally use frontmatter for individual page configuration.
+
+Frontmatter must be added to the top of a `.md` or `.mdx` file, before the rest of the content. Use sets of three dashes to indicate the beginning and end of your frontmatter, as shown below.
+
+<CodeBlock title="Example frontmatter in .md or .mdx file">
+```mdx
+---
+title: Fern Docs Quickstart
+description: Learn how to get started building a documentation site for your product with Fern. Automatically generate a API Reference using your OpenAPI specification, configure site navigation, customize styles, and write documentation in Markdown and MDX. Fern's magic puts it all together and builds an attractive site you can host yourself, or host free with Fern.
+editThisPageUrl: https://github.com/fern-api/fern/blob/main/fern/docs/pages/docs/quickstart.mdx
+image: https://github.com/fern-api/fern/blob/main/fern/docs/images/logo-green.png
+---
+```
+</CodeBlock>
+
+#### title
+**Type**: string<br />
+**Default**: The name of the page as specified in `docs.yml`
+
+Set the content for the [`<title>` element](https://web.dev/learn/html/document-structure#document_title) for a page. This title is displayed in browser tabs, history, and bookmarks, as well as in search engine results. Having a unique and informative title for each page benefits your site's SEO (Search Engine Optimization). For more information, see [Google's guidelines for useful titles](https://developers.google.com/search/docs/appearance/title-link#page-titles).
+
+If no title is provided in the frontmatter of a page, the name of the page in `docs.yml` is used.
+
+#### description
+**Type**: string<br />
+**Default**: None
+
+Set the [meta description](https://web.dev/learn/html/metadata#description) for a page. Like the page title, the meta description is important for SEO. It impacts the text that search engines display about your page in search results snippets. It can also influence search engine indexing and ranking. For more information, see [Google's guidelines for meta descriptions](https://developers.google.com/search/docs/appearance/snippet#meta-descriptions).
+
+#### editThisPageUrl
+**Type**: string (absolute URL)<br />
+**Default**: None
+
+Provide the absolute link to the source `.md` or `.mdx` file in GitHub. Fern uses it to add an `Edit this page` link to the page, which users of your documentation can use to suggest corrections or additions. 
+
+#### image
+**Type**: string (absolute URL)<br />
+**Default**: None
+
+Configure the `<meta property="og:image">` metadata for a page using an absolute path to an image hosted online. This provides an image to show next to a link to your documentation when the link is shared on social media, using a metadata protocol called [OpenGraph](https://ogp.me/). For more information, see the [web.dev explanation of OpenGraph](https://web.dev/learn/html/metadata#open_graph).
+
+### Links in Markdown
```

Comment:
We tend to see more questions about links. I think we should further flesh this out and include more examples of a relative link and an absolute link.